.github/workflows/qaqc.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         check_filenames: true
         check_hidden: true
         skip: '*.js,qaqc.yml'
-        ignore_words_list: slippy,trough,thw
+        ignore_words_file: .codespellignore
```

Comment:
@scottyhq - Note the change to use a file instead of a list in the action. Should make it easier for tutorial leads to edit and add words.